packages/calling-stateful-client/src/TestUtils.ts

```diff
@@ -275,6 +279,10 @@ export class MockTranscriptionCallFeatureImpl implements TranscriptionCallFeatur
   public isTranscriptionActive = false;
   public emitter = new EventEmitter();
   public isConsentRequired = false;
+  public isTeamsConsentRequired = false;
```

Comment:
is this in stable as well? we have a CC comment for this `calling-sdk-beta` if not

Reply:
that's a good point I will CC those lines as well! Thanks for bringing this up